ComponentKit/Core/Scope/CKComponentScopeFrame.h

```diff
@@ -34,4 +34,16 @@ struct CKComponentScopeFramePair {
 
 @property (nonatomic, strong, readonly) CKComponentScopeHandle *handle;
 
+struct _CKChildScopeFrameRet {
```

Comment:
This is somewhat inelegant, but I need to return more than just one of these items… thoughts?